app/scss/components/_button.scss

```diff
@@ -10,6 +10,7 @@
   text-decoration: none;
 
   &:hover {
-    background-color: darken($orange, 10%);
+    background-color: lighten($orange, 2%);
+    transition: background-color 500ms ease;
```

Comment:
Snuck in this change, but I know we plan to polish buttons in the future with some nicer animations. 

In general, I prefer that on hover buttons lighten rather than darken and that it's a subtle shift (especially here where contrast with the white text would become an issue). Darkening, to me, communicates "pressed", since in a skeuomorphic design a pressed button would get a shadow, so in my mind darkening is better suited to an "active" state. I didn't add an active state here, but we can address that with other button enhancements.